frontend/src/modules/icons/index.ts

```diff
@@ -1,3 +1,9 @@
+/**
+ * @summary A short one-line description for the file
```

Comment:
Can you include a description of:

1. How to use an icon provided by this file
2. When we want to add more icons in the future, how would we create the tsx file from Adobe XD, or from an SVG file which we might have downloaded from somewhere?

Reply:
Updating the comments as below

```
/**
 * @summary All icons (SVG) which are used in our App
 * 
 * To convert existing SVG icon to be compatible to our App,
 * SVG tag should be replaced with `SvgIcon` tag of Material
 * And also copy the props to override settings using `<SvgIcon {...props}` /> syntac
 * 
 * Example to use Icons in react compoennt
 * 
 * <ClockIcon/>
 * 
 * To override the settings for icon, use following syntax
 * 
 * <ClockIcon style={{ fontSize: '2.5rem' }} />
 * 
 */
```